### ISSUE TITLE
How can I get image.ptif file?

### ISSUE THREAD

**Issue by jeffstud** [astromatic/Leaflet.TileLayer.IIP#5]
HI,
I have made the iipimage server work. When I try to use leaflet plugin for iipimage server, it doesn't work. I think the problem may be here:
layer = L.tileLayer.iip('/fcgi-bin/iipsrv.fcgi?FIF=/path/to/image.ptif').addTo(map);
two questions:
1.Does FIF=/images/image.ptif equal to C:\images\image.ptif?
2.I have only tiled tif file, have not ptif file? How can I get it?
Thank you very much.

**Comment by jeffstud** [comment]
Can I rename image.tif to image.ptif?

**Comment by ebertin** [comment]
> Does FIF=/images/image.ptif equal to C:\images\image.ptif?

Unfortunately I am not familiar with Windows web servers. Is you web server running on Windows?

> I have only tiled tif file, have not ptif file? How can I get it?

IIPImage operates on tiled TIFF multiresolution "pyramids" which I chose to suffix with the ".ptif" extension. A simple TIFF file won't do it. You can create TIFF multiresolution pyramids in a number of ways (check out this [page](http://iipimage.sourceforge.net/documentation/images/). I can only advise you when it comes to creating TIFF multiresolutions pyramids from FITS, using the [STIFF package](http://www.astromatic.net/software/stiff): please check out the [STIFF documentation](https://www.astromatic.net/pubsvn/software/stiff/trunk/doc/stiff.pdf) or [this page](http://www.astromatic.net/software/visiomatic).

**Comment by jeffstud** [comment]
Yes. My web server runs on windows. 
Could you send me a ptif link, I download it and test? Thank you very much.

**Comment by ageblade** [comment]
> Does FIF=/images/image.ptif equal to C:\images\image.ptif?

It depends on how you configure it on your IIS. check [this](http://iipimage.sourceforge.net/documentation/server/) and [this](http://iipimage.sourceforge.net/2011/08/installing-iipimage-on-iis7) with focus on FILESYSTEM_PREFIX variable

Also you need to host the IIP server on the same domain if you are using '/fcgi-bin/.' without the full domain URL

**Comment by ruven** [comment]
> 1.Does FIF=/images/image.ptif equal to C:\images\image.ptif?

Yes. You should use forward slashes / for your FIF paths, however.

>  Could you send me a ptif link, I download it and test? Thank you very much.

You can find an example tiled multi-resolution tiff here: [http://iipimage.sourceforge.net/documentation/images/](http://iipimage.sourceforge.net/documentation/images/)

**Comment by jeffstud** [comment]
Yes. I have download the sample image from the site. However, the suffix of it is tif, not ptif.
It doesn't work.
If I use iipmooviewer, the .tif file can be displayed.
What's the reason?
: (

**Comment by jeffstud** [comment]
the codes of viewer is as following:

  link rel="stylesheet" href="leaflet.css" /
  script src="leaflet-src.js"/script
  script src="TileLayer.IIP.js"></script

   var map = L.map('map', {center: [0.0,0.0], zoom: 3}),
       layer = L.tileLayer.iip('/fcgi-bin/iipsrv.fcgi?FIF=/images/output.ptif').addTo(map);

**Comment by jeffstud** [comment]
`<html>
 <head>
  <link rel="stylesheet" href="leaflet.css" />
  <script src="leaflet-src.js"></script>
  <script src="TileLayer.IIP.js"></script>
 </head>
 <body>
  <div id="map" style="height:256px"></div>
  <script>
  	
   var map = L.map('map', {center: [0.0,0.0], zoom: 3}),
       layer = L.tileLayer.iip('/fcgi-bin/iipsrv.fcgi?FIF=/images/output.ptif').addTo(map);
  </script>
 </body>
</html>
`

**Comment by jeffstud** [comment]
> <html>
 <head>
  <link rel="stylesheet" href="leaflet.css" />
  <script src="leaflet-src.js"></script>
  <script src="TileLayer.IIP.js"></script>
 </head>
 <body>
  <div id="map" style="height:256px"></div>
  <script>
  	
   var map = L.map('map', {center: [0.0,0.0], zoom: 3}),
       layer = L.tileLayer.iip('/fcgi-bin/iipsrv.fcgi?FIF=/images/output.ptif').addTo(map);
  </script>
 </body>
</html>

**Comment by jeffstud** [comment]
I create output.ptif using ImageMagick
I put the output.tif in the C:\images\
But I still get the information "unexpected response from iip server /fcgi-bin/iipsrv.fcgi"
Could you give me some advice?
Thank you.

**Comment by jeffstud** [comment]
my server is apache.

**Comment by ruven** [comment]
See what it says in the iipsrv log file.

**Comment by jeffstud** [comment]
I cannot find the iipsrv log file.

**Comment by jeffstud** [comment]
I think there are two possibilities.
first, the wrong path.
second, the wrong file.  Could you offer me a right ptif image like the one in demo? http://image.iap.fr/iip/
I want to use the same code and the same image to test to find the answer.
